### PR TITLE
PACA-516: Add FI_DOC_CLM_CNTL_NUM and CARR_CLM_CNTL_NUM definitions

### PR DESCRIPTION
**Pull request by KyRobbins**
**JIRA Ticket:**
[PACA-516](https://jira.cms.gov/browse/PACA-516)

**User Story or Bug Summary:**

### What Does This PR Do?

Add new codesystem values to the bluebutton static site so they can be referenced (and resolve) by the BFD FHIR API to indicate partially adjudicated data.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

* The added code system definition files follow the existing standards and conventions
* The layout template is appropriate and follows existing conventions

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? **Yes** or **No**.
    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

* CMSgov/some_repo#42

### Submitter Checklist

I have gone through and verified that...:

* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: _layouts/basic.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    {% include footer.html %}
+
+    <script
+    src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+    integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g="
+    crossorigin="anonymous"></script>
+
+    <!-- Feather Icons -->
+    <script src="https://unpkg.com/feather-icons"></script>
```
Comment (ericbuckley) [inline _layouts/basic.html:23]: whats the feather library used for?

Comment (KyRobbins) [reply _layouts/basic.html:23]: This appears to be used to pull in SVG images to be used within the page layout.

**Review by ericbuckley — CHANGES_REQUESTED** [review]
based on the information we discovered today, both of these URLs will need to be under the `/resources/variables` path so they are backwards compatible with the coding systems already in use in the v1 EOBs

File: _pages/resources-variables-fi_doc_clm_cntl_num.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Description
+This field is an identification value for institutional claims processed by the fiscal intermediary standard system (FISS). It provides a reference number for the control and monitoring of specific claims, and to differentiate between claims for the beneficiary. This is a unique field created by FISS.
+
+## Other Info
+- **Short name:** FI_DOC_CLM_CNTL_NUM
+- **Long name:** FI_DOC_CLM_CNTL_NUM
+- **Type:** char
+- **Length:** 23
+- **Source:** FISS
```
Comment (ericbuckley) [inline _pages/resources-variables-fi_doc_clm_cntl_num.md:16]:
```suggestion
- **Source:** CCW
```

File: _pages/resources-variables-carr_clm_cntl_num.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Description
+This field is an identification number for professional claims processed by the multi-carrier system (MCS). This field contains a numeric value that identifies a claim throughout the processing cycle and allows for a linkage between the claim and the beneficiary for whom it was submitted. This should be handled as a character field due to leading ‘0’s. This is a unique field created by MCS.
+
+## Other Info
+- **Short name:** CARR_CLM_CNTL_NUM
+- **Long name:** CARR_CLM_CNTL_NUM
+- **Type:** char
+- **Length:** 15
+- **Source:** MCS
```
Comment (ericbuckley) [inline _pages/resources-variables-carr_clm_cntl_num.md:16]:
```suggestion
- **Source:** CCW
```

**Review by ericbuckley — APPROVED** [review]

**Review by brianburton — APPROVED** [review]

**Comment by micahtaylor** [comment]
Per discussion with @ericbuckley I added in the 404 page title to the 404 page as well to help with Google Analytics reporting.

File: _layouts/basic.html (hunk 1 above)
Comment (micahtaylor) [reply _layouts/basic.html:23]: Yup. Lib used for icons.

**Review by micahtaylor — APPROVED** [review]
Looks good. I will squash and merge and deploy on Monday.

**Pull request merged**